src/App.js

```diff
@@ -0,0 +1,46 @@
+import "./styles.css";
+import { Cart } from "./Cart/Cart";
+import { ProductsPage } from "./ProductsPage/ProductsPage";
+import { WishList } from "./WishList/WishList";
+import { HomePage } from "./HomePage/HomePage";
+import { Header } from "./HomePage/Header/Header";
+import { OneProductPage } from "./OneProductPage/OneProductPage";
+import { CardDetails } from "./Cart/Checkout/CardDetails/CardDetails";
+import { NoMatch } from "./NoMatch404/NoMatch";
+import { useThemeContext } from "./AllContext/themeContext";
+import { Routes, Route } from "react-router-dom";
+import { LoginPage } from "./LoginPage/LoginPage";
+import { PrivateRoute } from "./PrivateRoute";
+import { AddressManagement } from "./Cart/Checkout/AddressManagement/AddressManagement";
+import { Addresses } from "./ProfilePage/Addresses/Addresses";
+import { SignUpPage } from "./SignUpPage/SignUpPage";
+import { EachSellerPage } from "./EachSellerPage/EachSellerPage";
+import { Profile } from "./ProfilePage/Profile/Profile";
+import { Coupons } from "./ProfilePage/Coupons/Coupons";
```

Comment:
I have a suggestion here, to make it more clean add index.js file to each folder and export all components from there as well.
Also you can create components folder to keep all these folders there and add index.js in components folder too